administrator/components/com_content/src/Model/ArticleModel.php

```diff
@@ -469,6 +470,92 @@ public function getItem($pk = null)
 			}
 		}
 
+		// Expression to search for (positions)
+		$regex = '/{loadposition\s(.*?)}/i';
+
+		// Expression to search for (modules)
+		$regexmod = '/{loadmodule\s(.*?)}/i';
+
+		// Expression to search for (id)
+		$regexmodid = '/{loadmoduleid\s([1-9][0-9]*)}/i';
+
+		// Find all instances of plugin and put in $matches for loadposition
+		// $matches[0] is full pattern match, $matches[1] is the position
```

Comment:
https://developer.joomla.org/coding-standards/inline-code-comments.html
```suggestion
		/**
		 * Find all instances of plugin and put in $matches for loadposition
		 * $matches[0] is full pattern match, $matches[1] is the position
		 */
```